src/test/groovy/org/gradle/android/TaskAvoidanceTest.groovy

```diff
@@ -0,0 +1,55 @@
+package org.gradle.android
+
+import org.gradle.util.GradleVersion
+import org.gradle.util.VersionNumber
+import spock.lang.Unroll
+
+class TaskAvoidanceTest extends AbstractTest {
+    @Unroll
+    def "tasks are avoided with #gradleVersion and Android plugin #androidVersion"() {
+        assert gradleVersion instanceof GradleVersion
+        assert androidVersion instanceof VersionNumber
+
+        println "> Using Android plugin $androidVersion"
+        println "> Running with $gradleVersion"
+
+        SimpleAndroidApp.builder(temporaryFolder.root, cacheDir)
+                .withAndroidVersion(androidVersion)
+        .withKotlinDisabled()
```

Comment:
Polish indenting

Reply:
I wonder why codenarc - which was executed on CI on the test sources - did not cache this: https://scans.gradle.com/s/bkrkyiqc6bcrc/timeline?details=mx7qk7s42pldi&hideTimeline
Will fix.